app/models/grda_warehouse/shape/state_scopes.rb

```diff
@@ -9,8 +9,8 @@ module Shape
     module StateScopes
       extend ActiveSupport::Concern
       included do
-        scope :my_state, -> { where(statefp: my_fips_state_code) }
-        scope :not_my_state, -> { where.not(statefp: my_fips_state_code) }
+        scope :my_states, -> { where(statefp: my_fips_state_codes) }
```

Comment:
It might be helpful to make sure all the relevant tables index statefp, but I'm not sure if it's actually beneficial.

Reply:
Hmm, maybe, though with a max of 3 states right now, I'm not totally convinced.  Also, I _think_ we may join on the state column (maybe?), but we aren't limiting on it, we're just showing all of the relevant states.  Looks like we already have one on `shape_cocs`, `shape_states`, `shape_counties`, and `state_zip_codes` so mostly covered.  Let's try without the index and revisit if we have any performance issues.